salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc

```diff
@@ -0,0 +1,101 @@
+. Click *Save*.
+
+Copy the sales order ID, which you'll need in the next steps.
+
+== Create a Freight Order
+
+Follow these steps to create a freight order in SAP:
+
+. Search for the *Transportation Cockpit* app.
+. Select *ERP Document* and paste the sales order ID into the *Freight Unit Stages* field.
+. Enter *** in the *Vehicle Types* field.
+. Click *Continue*.
+. Select the freight unit of the sales order in the *Freight Unit* column.
+. Select a value in the *Trucks* field.
+. Click *Create Road Freight Order* in the *Freight Unit Stages* field, and then click *Save* to create the freight order.
+. Click the freight order number to open the order details, and then complete the *Vehicle Type*, *Carrier*, *Purchasing Organization*, and *Purchasing Group* fields.
+. On the *Charges* tab, enter an amount in the *Base FTL - Flat Rate* field, and then click *Save*.
+. From the *Subcontracting* dropdown on the *Statuses* tab, select *Send to Carrier*.
+. Then, from the same *Subcontracting* dropdown, select *Set to Confirmed by Carrier*.
+. Click *Custom Fields* from the *All Tabs* menu icon (image:all-tabs-icon.png[3%,3%]) and complete the *Tracking Number* field.
+. Click *Save*.
+
+== Create an Outbound Delivery
+
+Follow these steps to create an outbound delivery in SAP:
+
+. Click the *Create Deliveries* tab, and then click the *Document Flow* tab.
+. Check the box for each document that is related to the freight order.
+. Open SAP S/4HANA in a new tab and search for the *Change Outbound Delivery* app.
+. Enter a value in the *Outbound Delivery* field, and then press Enter.
+. On the *Picking* tab, enter the same value in the *Picked Qty* column as the value in the *Deliv. Qty* column. Entering this value updates the status values in the *OvrllPickStatus* and *Warehouse Execution* fields.
+. Select *Post Goods Issue* from the *Menu* dropdown. This action triggers the `Outbound Delivery API (DELIVERY_REQUEST_OUT)` event.
+. Navigate back to the freight order and click *Refresh* to update the status fields on the *Statuses* tab.
+. Click the *Edit* button.
+. Select *Set to Ready for Transportation Execution* in the *Execution Status* dropdown. This updates the value in the *Execution Status* field to *Ready for Transportation Execution*.
+. In the *Stages* section, select *Set to Departed* in the *Execution Status (Source)* dropdown.
+
+== Create an Invoice
+
+Follow these steps to create an invoice in SAP:
+
+. Search for the *Create Billing Documents - VF01* app.
+. Enter the outbound delivery number in the *Document* column of the *Documents to Be Processed* table, and then press Enter.
+. Click *Save*.
+
+== View Your Documents
+
+Follow these steps in SAP to view the documents you created:
+
+. Search for the *Manage Sales Order Version 2* app, and then filter by the sales order ID.
+. Click the sales order ID in the *Sales Orders* table to open the order details.
+. Click *Process Flow* to view the documents you created.
+
```

Comment:
@nisha-nizamuddin Do we want to include a **Next Steps** section to let the user know where to go once they've completed the steps in this guide?